governance-test-scripts/src/main.rs

```diff
@@ -37,18 +37,18 @@ use governance_lib::{
     addin_fixed_weights::AddinFixedWeights,
 };
 
-const GOVERNANCE_KEY_FILE_PATH: &str = "solana-program-library/target/deploy/spl_governance-keypair.json";
-const VOTER_WEIGHT_ADDIN_KEY_FILE_PATH: &str = "target/deploy/spl_governance_addin_fixed_weights-keypair.json";
-const COMMUTINY_MINT_KEY_FILE_PATH: &str = "governance-test-scripts/community_mint.keypair";
-const GOVERNED_MINT_KEY_FILE_PATH: &str = "governance-test-scripts/governance.keypair";
-const PAYER_KEY_FILE_PATH: &str = "artifacts/payer.keypair";
+const GOVERNANCE_KEY_FILE_PATH: &str = "../artifacts/spl_governance-keypair.json";
+const VOTER_WEIGHT_ADDIN_KEY_FILE_PATH: &str = "../artifacts/addin-fixed-weights.keypair";
+const COMMUTINY_MINT_KEY_FILE_PATH: &str = "../governance-test-scripts/community_mint.keypair";
+const GOVERNED_MINT_KEY_FILE_PATH: &str = "../governance-test-scripts/governance.keypair";
+const PAYER_KEY_FILE_PATH: &str = "../artifacts/payer.keypair";
 
 const VOTERS_KEY_FILE_PATH: [&str;5] = [
-    "artifacts/voter1.keypair",
-    "artifacts/voter2.keypair",
-    "artifacts/voter3.keypair",
-    "artifacts/voter4.keypair",
-    "artifacts/voter5.keypair",
+    "../artifacts/voter1.keypair",
+    "../artifacts/voter2.keypair",
+    "../artifacts/voter3.keypair",
+    "../artifacts/voter4.keypair",
+    "../artifacts/voter5.keypair",
```

Comment:
Why do you need to change these paths? I suggest running governance-test-scripts from the root repository directory.